kaldb/src/main/java/com/slack/kaldb/util/SnapshotUtil.java

```diff
@@ -4,6 +4,6 @@
 
 public class SnapshotUtil {
```

Comment:
This should be moved to the test package, as this is only for use in tests and not production code.

Reply:
Good call. Moved.